examples/Registration/README.txt

```diff
@@ -0,0 +1,7 @@
+data from Slicer's BRAINSTools Test Data (http://midas.kitware.com/item/view/483)

```

Comment:
convert to README.md. add a sentence or 2 about general purpose of this directory